diff --git a/debian/rules b/debian/rules
index 80a8e75..6a41c6e 100755
--- a/debian/rules
+++ b/debian/rules
@@ -1,14 +1,23 @@
 #!/usr/bin/make -f
-export DH_VERBOSE=1
-export METALBOND_VERSION=$(dpkg-parsechangelog -SVersion)
+# standard Debian options & profiles

-%:
-	dh $@
+export DEB_BUILD_MAINT_OPTIONS = hardening=+all

+ifneq (,$(filter terse,$(DEB_BUILD_OPTIONS)))
+  MAKE_SILENT="V=0"
+  export DH_VERBOSE=0
+else
+  MAKE_SILENT="V=1"
+  export DH_VERBOSE=1
+  export DH_OPTIONS=-v
+endif

-overwrite_dh_auto_build:
-	echo "### debug $(METALBOND_VERSION)"
-	make

+%:
+	dh $@

+source:
+	:

+override_dh_dwz:
+	:
